libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -184,26 +208,27 @@ While the text is visually hidden, it is still remains accessible.
 <DocCanvas client:only
   mdxSource={{
     react: `
-    <PdsButton variant="primary" iconOnly="true" icon="favorite">Icon Only</PdsButton>
-    <PdsButton variant="secondary" iconOnly="true" icon="favorite">Icon Only</PdsButton>
-    <PdsButton variant="accent" iconOnly="true" icon="favorite">Icon Only</PdsButton>
-    <PdsButton variant="destructive" iconOnly="true" icon="favorite">Icon Only</PdsButton>
+    <PdsButton variant="primary" iconOnly="true"><pds-icon slot="start" aria-hidden="true" name="favorite"></pds-icon>Icon Only
+    </PdsButton>
+    <PdsButton variant="secondary" iconOnly="true"><pds-icon slot="start" aria-hidden="true" name="favorite"></pds-icon>Icon Only
+    </PdsButton>
+    <PdsButton variant="accent" iconOnly="true"><pds-icon slot="start" aria-hidden="true" name="favorite"></pds-icon>Icon Only</PdsButton>
+    <PdsButton variant="destructive" iconOnly="true"><pds-icon slot="start" aria-hidden="true" name="favorite"></pds-icon>Icon Only</PdsButton>
```

Comment:
Can you change the web component version of `pds-icon` to use the React version `PdsIcon`?

Reply:
Done